### ISSUE TITLE
Contributor capes

### ISSUE THREAD

**Issue by SimonMeskens** [BTW-Community/BTW-Public#349]
Would be kinda cool if we added special capes for contributors to CE

**Comment by ammoore00** [comment]
Yeah I'd be down. We'd probably want some way to configure it so people can display an official Mojang cape instead of a dev cape if they want. Would we want a separate server maintaining the list of contributors, or just hardcode usernames into releases? The latter would definitely be easier, but it'd be nice if we didn't have to put out a release to update the lsit.

**Comment by SimonMeskens** [comment]
We probably need to look at the whole thing while fixing #348 anyway. Might as well do it right and have a Github repo with the names or something.
